cypress/support/utils/apiUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { addCandidatePayload } from "../API_Helpers/payloads/addCandidatePayload";
+import { addCandidateResponse } from "../API_Helpers/responses/addCandidateResponse";
+
+
+declare global {
+    namespace Cypress {
+        interface Chainable {
+            addNewCandidate: (requestURL: string, candidatePayload: addCandidatePayload) => Chainable<addCandidateResponse>;
+        }
+    }
+
+}
+
+
+Cypress.Commands.add('addNewCandidate', (requestURL: string, candidatePayload: addCandidatePayload) => {
```

Comment:
why in a command? do it as a request in the POM, we do commands only when it's commonly used.